tasks.py

```diff
@@ -190,6 +192,23 @@ def open_editor(initial_message):
     return lines
 
 
+def check_prerequisites():
+    result = invoke_run("which twine", hide=True)
```

Comment:
`which` isn't very portable: a more Pythonic alternative might be:
```python
from distutils.spawn import find_executable


if find_executable("twine") is None:
    error(
        "twine executable not found. "
        "You must have twine to run this command."
    )
    exit(127)
```

Reply:
ah nice, thank you!